dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -676,6 +707,39 @@ protected void loadData(String ufsPath, long mountId, long offset, long lengthTo
     }
   }
 
+  /**
+   * Loads data from remote worker.
+   *
+   * @param filePath     the file path
+   * @param offset       the offset
+   * @param lengthToLoad the length to load
+   * @param reader       the netty reader
+   * @param chunkSize    the chunk size
+   * @throws IOException when failed to read from remote worker
+   */
+  @VisibleForTesting
+  public void loadDataFromRemote(String filePath, long offset, long lengthToLoad,
+      PositionReader reader, int chunkSize) throws IOException {
+    ByteBuffer buf = ByteBuffer.allocate(chunkSize);
```

Comment:
use PooledDirectNioByteBuf.allocate(chunkSize) as loadData does since most of it will be aligned to pagesize, and pass in NettyBufTargetBuffer type for reader.read(long position, ReadTargetBuffer buffer, int length) as this pool can manage a reuse of these mostly aligned buffer. otherwise this is allocate onheap will cause huge mem footprint and put heavylifting on GC

Reply:
oh u use ByteBuffer bcos of cachemanager only support bytebuffer, then its better to allocate direct buffer than heap, large buffer mem footprint on heap might cause program misbehavior.

Reply:
use direct buffer